controllers/database_reconciler_helpers.go

```diff
@@ -154,11 +168,15 @@ func (r *DatabaseReconciler) handleExternalDelete(ctx context.Context, database
 		}
 		// Update the CR status if the database response is empty so that it triggers a provision operation
 		if databaseResponse.Status == common.DATABASE_CR_STATUS_EMPTY {
-			log.Info("The database might have been deleted externally, setting an empty status so it can be re-provisioned.")
+			infoStatement := "The database might have been deleted externally, setting an empty status so it can be re-provisioned."
```

Comment:
will this msg be user-facing? or mostly for our debugging?

Reply:
IMO, this should not be user facing. @shenoypritika can we use something like "Retrying provisioning the database" for the event?